src/org/roana0229/android_xml_sorter/XmlSorterAction.java

```diff
@@ -56,8 +55,25 @@ public void actionPerformed(AnActionEvent event) {
         boolean enableInsertXmlEncoding = dialog.enableInsertXmlInfo();
         boolean enableDeleteComment = dialog.enableDeleteComment();
         int codeIndent = dialog.getCodeIndent();
+        execute(event,
+                isSnakeCase,
+                prefixSpacePosition,
+                enableInsertSpaceDiffPrefix,
+                enableInsertXmlEncoding,
+                enableDeleteComment,
+                codeIndent);
+    }
 
+    protected void execute(AnActionEvent event,
+                           boolean isSnakeCase,
+                           int prefixSpacePosition,
+                           boolean enableInsertSpaceDiffPrefix,
+                           boolean enableInsertXmlEncoding,
+                           boolean enableDeleteComment,
+                           int codeIndent) {
         // get content
+        final Project project = getEventProject(event);
```

Comment:
error was displayed at the lower right(! mark), when `meta + L` was used.

```
cannot share data context between Swing events; initial event count = 1763; current event count = 1902
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:132)
	at com.intellij.ide.impl.DataManagerImpl$MyDataContext.getData(DataManagerImpl.java:363)
	at com.intellij.openapi.actionSystem.DataContext.getData(DataContext.java:59)
	at com.intellij.openapi.actionSystem.AnActionEvent.getData(AnActionEvent.java:165)
	at com.intellij.openapi.actionSystem.AnAction.getEventProject(AnAction.java:330)
	at org.roana0229.android_xml_sorter.XmlSorterAction.execute(XmlSorterAction.java:75)
	at org.roana0229.android_xml_sorter.XmlSorterAction.actionPerformed(XmlSorterAction.java:58)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:216)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:233)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:576)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$2(IdeKeyEventDispatcher.java:625)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:193)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:624)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:479)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:213)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:621)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Reply:
error don't display when I changed `execute(AnActionEvent event, ...` to `execute(Project project, Editor editor, ...`.

Can you confirm and change this?